src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/random/RandomUtilitiesTest.kt

```diff
@@ -63,8 +65,7 @@ class RandomUtilitiesTest {
             )
           )
         ),
-        has(Recall::recallType, present()),
-        has(Recall::recallLength, present()),
```

Comment:
Wondering if there's some conditional Matcher usage we can replace `has(Recall::recallLength, present())` with, rather than drop it completely?

Reply:
Yes, did cross my mind. any ideas 😬 ?